scripts/__dropins__/storefront-company-management/hooks/index.d.ts

```diff
@@ -15,5 +15,6 @@
  * from Adobe.
  *******************************************************************/
 export * from './containers';
+export * from './useCompanyContextListener';
```

Comment:
This chunk was missing. Remember to `npm install` after you install/update the dropin. And commit ALL the new scripts files.